test/conformance/api/shared/util.go

```diff
@@ -119,16 +125,17 @@ func checkResponses(t pkgTest.TLegacy, num, min int, domain string, expectedResp
 	//   GROUP BY body
 	// )
 	for _, ar := range actualResponses {
```

Comment:
Isn't the index here (if you not use underscore but `i`) exactly the same index? Would spare you the extra type and all.